src/middlewared/middlewared/plugins/service.py

```diff
@@ -934,6 +934,7 @@ def _started_notify(self, verb, what):
     async def _start_cifs(self, **kwargs):
         await self._service("ix-pre-samba", "start", quiet=True, **kwargs)
         await self._service("samba_server", "start", quiet=True, **kwargs)
+        await self.middleware.call("smb.add_admin_group", "", True)
```

Comment:
Is this persistent?

Or in other words, dont we need to set that on boot?

Reply:
It is persistent.